packages/angular-cli/tasks/npm-install.ts

```diff
@@ -0,0 +1,26 @@
+const Task = require('ember-cli/lib/models/task');
+import * as chalk from 'chalk';
+import {exec} from 'child_process';
+
+
+export default Task.extend({
+  run: function () {
+    const ui = this.ui;
+    let exitCode = 0;
+
+    return new Promise((resolve) => {
+      ui.writeLine(chalk.green('Installing packages for tooling via npm.'));
+      exec(`npm install`,
+        (err: NodeJS.ErrnoException, stdout: string, stderr: string) => {
+          if (err) {
+            ui.writeLine(stderr);
+            ui.writeLine(chalk.red('Package install failed, see above.'));
+            exitCode = 1;
```

Comment:
Fixed here but not the e2e. I assume it needs further changes and don't wanna mix the two things up.